### PR TITLE
Adding suggestion from Jaime and napari version Note

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
# Description
I've added the suggestion from Jaime
https://github.com/napari/docs/pull/51#discussion_r1026208208
And the Note about the version.
I fixed the GitHub action to use the proper version. I'm not sure if the other actions also need changing.

YOu can check the build docs in my fork PR:
https://github.com/psobolewskiPhD/napari-docs/pull/3

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -158,6 +159,12 @@
 
 myst_heading_anchors = 3
 
+version_string = '.'.join(str(x) for x in __version_tuple__[:3])
+
+myst_substitutions = {
+   "napari_version": version_string,
```
Comment (Czaki) [inline docs/conf.py:166]:
```suggestion
   "napari_conda_version": f"`napari={version_string}`",
```

File: docs/tutorials/fundamentals/installation.md [hunk 2]
```diff
@@ -120,12 +120,11 @@ conda install -c conda-forge "napari=*=*pyside2"
...
-It can happen that `conda`'s built-in solver is not able to solve all denpendencies needed to install `napari`. This can happen even in a clean environment
-depending on your operating system. In this case use `pip` for the installation or if that is no option, install `mamba` and use 
-it in place of `conda` to install from `conda-forge`. For example, to install napari with the pyside2 Qt backend:
-```sh
-mamba install -c conda-forge "napari=*=*pyside2"
-```
+In some cases, `conda`'s default solver can struggle to find out which packages need to be
+installed for napari. If it takes too long or you get the wrong version of napari 
+(see below), consider:
+1. Overriding your default channels to use only `conda-forge` by adding `--override-channels` and specifying the napari and Python versions explicitly using `python=3.9` to get Python 3.9 and `napari=0.x.y` to specify the napari version, such as the current release {{ napari_version }}.
```
Comment (Czaki) [inline docs/tutorials/fundamentals/installation.md]:
```suggestion
1. Overriding your default channels to use only `conda-forge` by adding `--override-channels` and specifying the napari and Python versions explicitly using `python=3.9` to get Python 3.9 and {{ napari_conda_version }} to specify the napari version.
```

**Review by Czaki — CHANGES_REQUESTED** [review]
I think that it could be done better. We cannot substitute in the code block, but we could substitute the whole code block. 

obraz

File: docs/conf.py (hunk 1 above)
Comment (psobolewskiPhD) [reply docs/conf.py:166]: Whoops, that should have been added (+)
Fix incoming.

**Comment by psobolewskiPhD** [comment]
There was a small bug after merging the fix from Czaki where the regular substitution didn't work. I've fixed it. Here's the built docs now:
image
image
Can access here:
https://github.com/psobolewskiPhD/napari-docs/actions/runs/3610147414

**Comment by Czaki** [comment]
Maybe also specify the python version as a variable in `conf.py` to have same python version everywhere in docs?

**Comment by psobolewskiPhD** [comment]
> Maybe also specify the python version as a variable in `conf.py` to have same python version everywhere in docs?

Good idea. I won't be able to get to it till at least Tuesday—traveling the next two days.
Also, maybe it's a bit of scope creep—which admittedly even this PR to the original PR is somewhat—and should be it's own PR?

**Comment by Czaki** [comment]
but in the scope of this PR may be introduced this variable and use, and followup could be to reintroduce it in another place.

**Pull request merged**